community/mpv/APKBUILD

```diff
@@ -5,21 +5,49 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=mpv
 pkgver=0.29.1
-pkgrel=1
+pkgrel=2
```

Comment:
Are there any function changes, or is it just style changes? In the case of the latter, a `pkgrel` bump is not necessary (no need to rebuild the packages)

Reply:
Or is it necessary to rebuild because of dependencies?

Reply:
yes, rebuild needed due to the new dependencies.